source/Octopus.Tentacle.Tests.Integration/Support/SetupFixtures/BumpThreadPoolForAllTests.cs

```diff
@@ -9,8 +9,8 @@ public class BumpThreadPoolForAllTests : ISetupFixture
         public void OneTimeSetUp(ILogger logger)
         {
             logger.Information("Bumping thread pool");
-            var minWorkerPoolThreads = 5000;
-            var minCompletionPortThreads = 5000;
+            var minWorkerPoolThreads = 1000;
```

Comment:
We have observed in testing that having a min thread pool that is too large can hurt performance as .net struggles to manage the scheduling of the threads.

I observed similar issues locally where a test would just freeze for a long duration and then fail. Reducing this as a possible contributor